src/main/java/org/itstep/j2_16/entity/Employee.java

```diff
@@ -0,0 +1,24 @@
+package org.itstep.j2_16.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+@Data
+@NoArgsConstructor
+public class Employee {
+    @Id
+    @GeneratedValue
+    private long id;
+    @Column
+    private String firstName;
+    private String lastName;
+    private String sex;
+    private int age;
```

Comment:
Plz, do not use primitive types for entity fields. I'll explain it later, just remember me during the lesson